UnityProject/Assets/Scripts/Player/IPlayerPossessable.cs

```diff
@@ -106,18 +136,25 @@ public void InternalOnEnterPlayerControl(GameObject previouslyControlling, Mind
 			ClientInternalOnEnterPlayerControl(previouslyControlling, mind, isServer);
 		}
 
-		OnEnterPlayerControl( previouslyControlling,  mind,  isServer);
+		OnEnterPlayerControl(previouslyControlling, mind, isServer, parent);
+		var Possessing = GetPossessing();
```

Comment:
```suggestion
		var possessing = GetPossessing();
```
the name you chose is even shadowing a member of this class.